migration/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.axonframework</groupId>
+        <artifactId>axon</artifactId>
+        <version>4.7.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>migration</artifactId>
+
+    <name>Axon Framework - Migration</name>
+    <description>
+        Module containing the migration recipes.
+    </description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.axonframework</groupId>
+            <artifactId>axon-messaging</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openrewrite.recipe</groupId>
+            <artifactId>rewrite-spring</artifactId>
```

Comment:
Curious: Why have this `rewrite-spring` dependency here?

Reply:
That's in case you want to dasiy-chain migrations for Spring (optional) or any of the recipes it includes, such as jakarta, Java 17, JUnit, ...

Reply:
Awesome, thanks for the explanation.